app/views/user_mailer/creatorship_notification.text.erb

```diff
@@ -1,14 +1,12 @@
 <% creation_type = @creation.model_name.human.downcase %>
 <% content_for :message do %>
-<%= t(".intro", type: creation_type, adding_user: @adding_user.login,
-      pseud: @creatorship.pseud.name) %>
+  <%= t(".intro_#{creation_type}", adding_user: @adding_user.login, pseud: @creatorship.pseud.name) %>
 
-<%= t ".text.creation", title: creation_title(@creation),
-  url: polymorphic_url(@creation),
-  pseuds: @creation.pseuds.map(&:byline).to_sentence %>
+  <%= t ".text.creation", title: creation_title(@creation),
```

Comment:
Whoops, I just realized: can you remove all the indenting you added to the text emails? Translation pointed out it actually causes the text in the email to be indented.

Reply:
OK, fixing